requirements.txt

```diff
@@ -0,0 +1,6 @@
+matplotlib==3.8.0
+networkx==3.1
+numpy==1.26.1
+nx-parallel==0.1
+pandas==2.1.1
+seaborn==0.13.0

```

Comment:
These requirements are for the timing scripts, not for the package `nx-parallel`. The dependencies of the package are in the pyproject.toml file.